src/components/Sidebar.jsx

```diff
@@ -0,0 +1,29 @@
+import { NavLink } from "react-router-dom"
+import { BsTwitter } from "react-icons/bs"
+import { FaFacebookF } from "react-icons/fa"
+import { TiSocialGooglePlus } from "react-icons/ti"
+import { BsVimeo } from "react-icons/bs"
+import { FaPinterestP } from "react-icons/fa"
+
+const Sidebar = () => {
+  return (
+    <div className="sidebar">
+      <h1>Zuku Apartments</h1>
+      <div id="links-div">
+        <NavLink className="nav-link" to="/example">EXAMPLE</NavLink>
+        <NavLink className="nav-link" to="/test">TEST</NavLink>
+        <NavLink className="nav-link" to="/">HOME</NavLink>
+      </div>
```

Comment:
Of course, this is the beginning of the project, but it is also important to avoid unnecessary work and waste of time by adding the right texts on the sidebar.
`Houses` - `Add House` - `My Reservations` - `Add Reservation`